### DISCUSSION TITLE
Shows strength and not HP during combat

### DISCUSSION THREAD

**Discussion by Hulksters** [runelite/runelite#18227] [Q&A]
I just downloaded Runelite and loving it! During combat though, I am not seeing the status widget displayed on the website. I do see a window over the enemy showing its remaining hit points. Over my character, it shows strength and not HP remaining. Is there a way to change that?

**Comment by Hulksters** [comment]
I think I figuired some of this out. Above my character is the stats required to level up the stat associated with my combat style. So, I had strength combat selected....so it showed strength needed for the next level. Got it!

So, my question is how do I see the attached status widget?

![Capture](https://github.com/user-attachments/assets/bf7910c2-2304-4a2e-becf-e76df2a8cf10)

**Reply by cwjoshuak** [reply]
The image you posted consists of multiple plugins. 

The hp one being [Opponent Information](https://github.com/runelite/runelite/wiki/Opponent-Information) and the infoboxes are probably [Boosts Information](https://github.com/runelite/runelite/wiki/Boosts-Information) and maybe [Slayer](https://github.com/runelite/runelite/wiki/Slayer) and Timers.
All of which you can find and customize in the settings sidepanel.

**Comment by Hulksters** [comment]
much thanks!!!!!!!!!!!
